blocks/components/editable/index.js

```diff
@@ -233,7 +233,7 @@ export default class Editable extends wp.element.Component {
 		}
 
 		content = wp.element.renderToString( content );
-		this.editor.setContent( content );
+		this.editor.setContent( content, { format: 'raw' } );
```

Comment:
I think each time we use an `Editable` we'd want to provide `valid_elements` to say which tags, attributes etc... this Editable allows (We'll need good defaults). if we use TinyMCE's `valid_elements` setting for this, there's not much left for us to do.

But I can be convinced that the content should have already been sanitised in the block attributes and I'm thinking, we could offer a helper to do this sanitization outside of TinyMCE.